### PR TITLE
macOS fix Toolbar show/hide on macOS Mojave

### PR DESCRIPTION
**Pull request by sweetppro**
since refactoring to use Autolayout, systems using the older style **Expanded** NSToolbar have issues when Showing/Hiding the toolbar. This PR addresses that

### CODE REVIEW

**Comment by ckerr** [comment]
code-style failing on:

```diff
diff --git a/macosx/AboutWindowController.mm b/macosx/AboutWindowController.mm
index 417adf5..b69aec1 100644
--- a/macosx/AboutWindowController.mm
+++ b/macosx/AboutWindowController.mm
@@ -38,10 +38,11 @@ AboutWindowController* fAboutBoxInstance = nil;
     self.fCopyrightField.stringValue = [NSBundle.mainBundle localizedStringForKey:@"NSHumanReadableCopyright" value:nil
                                                                             table:@"InfoPlist"];
 
-    NSAttributedString *credits = [[NSAttributedString alloc] initWithURL:[NSBundle.mainBundle URLForResource:@"Credits" withExtension:@"rtf"]
-                                                                  options:@{NSDocumentTypeDocumentAttribute: NSRTFTextDocumentType}
-                                                       documentAttributes:nil
-                                                                    error:nil];
+    NSAttributedString* credits = [[NSAttributedString alloc]
+               initWithURL:[NSBundle.mainBundle URLForResource:@"Credits" withExtension:@"rtf"]
+                   options:@{ NSDocumentTypeDocumentAttribute : NSRTFTextDocumentType }
+        documentAttributes:nil
+                     error:nil];
     [self.fTextView.textStorage setAttributedString:credits];
 
     //size license button
```

**Comment by Coeur** [comment]
Comments added to the issue thread #3328 since I'm not reproducing the issue.

File: macosx/CMakeLists.txt [hunk 1]
```diff
@@ -95,6 +95,7 @@ set(${PROJECT_NAME}_SOURCES
     StatsWindowController.mm
     StatusBarController.mm
     StatusBarView.mm
+    Toolbar.mm
```
Comment (nevack) [inline macosx/CMakeLists.txt:98]: Need to add header also?

Comment (sweetppro) [reply macosx/CMakeLists.txt:98]: oops
added

**Review by nevack — APPROVED** [review]

**Comment by GaryElshaw** [comment]
Just noticed this popping up in the build process on CI during the QuickLook build process and didn't recognise this lib appearing previously at all.

```
error: /Applications/Xcode_13.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/install_name_tool: can't open file: /Users/runner/work/transmission/transmission/pfx/Applications/Transmission.app/Contents/MacOS/libicuuc.70.dylib (No such file or directory)
warning: target '/Users/runner/work/transmission/transmission/pfx/Applications/Transmission.app/Contents/MacOS/libicuuc.70.dylib' does not exist...
```

**Pull request merged**